doc/examples/animations/lib/main.dart

```diff
@@ -16,14 +17,24 @@ void main() async {
 }
 
 class MyGame extends BaseGame with TapDetector {
-  final animation = SpriteAnimation.sequenced(
-    'chopper.png',
-    4,
-    textureWidth: 48,
-    textureHeight: 48,
-    stepTime: 0.15,
-    loop: true,
-  );
+  Image chopper;
+  Image creature;
+  SpriteAnimation animation;
+
+  @override
+  Future<void> onLoad() async {
+    chopper = await Flame.images.load('chopper.png');
```

Comment:
should we update this example to use game.images as well?

Reply:
Sure thing! thanks for noticing that, done!